src/main/java/com/synopsys/integration/jenkins/scan/service/scm/BitbucketRepositoryService.java

```diff
@@ -84,4 +70,27 @@ public Bitbucket fetchBitbucketRepositoryDetails(Jenkins jenkins, Map<String, Ob
         return bitbucket;
     }
 
+    public static Bitbucket createBitbucketObject(String serverUrl, String bitBucketToken, Integer projectRepositoryPullNumber, String repositoryName, String projectKey) {
+        Bitbucket bitbucket = new Bitbucket();
+        Api bitbucketApi = new Api();
+        bitbucketApi.setUrl(serverUrl);
+        bitbucketApi.setToken(bitBucketToken);
```

Comment:
We can use these like - ` bitbucket.getApi().setUrl(serverUrl)`

Reply:
Thank you for the suggestion.